packages/camera/camera_windows/example/windows/CMakeLists.txt

```diff
@@ -0,0 +1,100 @@
+cmake_minimum_required(VERSION 3.14)
```

Comment:
I don't think it's necesary to do for this PR, but once this has landed we should probably regenerate with the latest template. There have been some (minor) changes, mostly just comments, recently.